src/dialects/abstract/connection-manager.d.ts

```diff
@@ -36,4 +36,10 @@ export interface ConnectionManager {
    * Destroys a pooled connection and removes it from the pool.
    */
   destroyConnection(conn: Connection): Promise<void>;
+
+  /**
+   * to be more strict, we can add type: {[key: string]: unknown}
+   * should we do it ?
+   */
+  pool: object;
```

Comment:
I'll need to take a look at the actual code but don't we define specific properties in the code of the dialects? Allowing any string as a key does not seem like much added value

Reply:
yeah makes sense actually, removing the comment

Reply:
"don't we define specific properties in the code of the dialects" - can you elaborate on this @WikiRik